lib/command.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class rex_developer_command extends rex_console_command
+{
+    protected function configure()
+    {
+        $this
+            ->setName('developer:sync')
+            ->setDescription('Synchronizes the developer files')
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $io = $this->getStyle($input, $output);
+
+        $io->title('Developer Sync');
+
+        rex_developer_manager::start();
```

Comment:
ggf. fliegende exceptions werden in der console selbst behandelt/dargestellt, oder?